### PR TITLE
Module1 cycles task1 (The sum of natural numbers)

### PR DESCRIPTION
**Pull request by pmiakish**

### CODE REVIEW

File: module_1/src/com/epam/cycles/CyclesTaskA.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+        if (a >= 1) {
+            int sum = 0;
+                for (int i = 1; i <= a; i++) {
+                    sum += i;
+                }
+            System.out.println("Total amount is " + sum);
+        } else {
+            System.out.println("The number must be greater than or equal to 1!");
+        }
```
Comment (lovelyserafima) [inline module_1/src/com/epam/cycles/CyclesTaskA.java:20]: this block of code should be on the same level as previos. Delete useless tabulation.

File: module_1/src/com/epam/cycles/CyclesTaskA.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+    Scanner console = new Scanner(System.in);
+    System.out.print ("Enter the natural number A (non-negative integer greater than or equal to 1): ");
+    int a = console.nextInt();
+
+        if (a >= 1) {
+            int sum = 0;
+                for (int i = 1; i <= a; i++) {
+                    sum += i;
+                }
```
Comment (lovelyserafima) [inline module_1/src/com/epam/cycles/CyclesTaskA.java:16]: And for-block also should be on the same level

File: module_1/src/com/epam/cycles/CyclesTaskA.java [hunk 3]
```diff
@@ -0,0 +1,23 @@
+package com.epam.cycles;
+import java.util.Scanner;
+
+public class CyclesTaskA {
+
+    public static void main(String[] args) {
+
+    Scanner console = new Scanner(System.in);
+    System.out.print ("Enter the natural number A (non-negative integer greater than or equal to 1): ");
```
Comment (lovelyserafima) [inline module_1/src/com/epam/cycles/CyclesTaskA.java:9]: delete the space after method System.out.print:
```suggestion
    System.out.print("Enter the natural number A (non-negative integer greater than or equal to 1): ");
```

File: module_1/src/com/epam/cycles/CyclesTaskA.java [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+    int a = console.nextInt();
+
+        if (a >= 1) {
+            int sum = 0;
```
Comment (lovelyserafima) [inline module_1/src/com/epam/cycles/CyclesTaskA.java:13]: let's change the type of variable from "int" to "long" to make the possibility to count bigger numbers
```suggestion
    long a = console.nextLong();

        if(a >= 1) {
            long sum = 0;
```

File: module_1/src/com/epam/cycles/CyclesTaskA.java [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+
+    if (a >= 1) {
+        int sum = 0;
+        for (int i = 1; i <= a; i++) {
+            sum += i;
+        }
+        System.out.println("Total amount is " + sum);
+    } else {
+        System.out.println("The number must be greater than or equal to 1!");
+    }
```
Comment (lovelyserafima) [inline module_1/src/com/epam/cycles/CyclesTaskA.java:20]: I'm sorry, maybe I told you wrong or you didn't catch me, but all this higlighted code should be shifted to tab to the right. I hope you understood me :)

File: module_1/src/com/epam/cycles/CyclesTaskA.java [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+public class CyclesTaskA {
+
+    public static void main(String[] args) {
+
+    Scanner console = new Scanner(System.in);
+    System.out.print("Enter the natural number A (non-negative integer greater than or equal to 1): ");
+    int a = console.nextInt();
+
+    if (a >= 1) {
+        int sum = 0;
```
Comment (lovelyserafima) [inline module_1/src/com/epam/cycles/CyclesTaskA.java:13]: I see here still integer variables
```suggestion
    long a = console.nextInt();

    if (a >= 1) {
        long sum = 0;
```

Comment (pmiakish) [reply module_1/src/com/epam/cycles/CyclesTaskA.java:13]: yes, I'm sorry, I missed this suggestion

**Pull request merged**